test/unit/util/FetchUtil.test.ts

```diff
@@ -1,52 +1,69 @@
+import type { Quad } from '@rdfjs/types';
 import arrayifyStream from 'arrayify-stream';
 import { fetch } from 'cross-fetch';
 import { DataFactory } from 'n3';
+import rdfDereferencer from 'rdf-dereference';
 import { RdfToQuadConverter } from '../../../src/storage/conversion/RdfToQuadConverter';
-import { fetchDataset } from '../../../src/util/FetchUtil';
+import { fetchDataset, responseToDataset } from '../../../src/util/FetchUtil';
 const { namedNode, quad } = DataFactory;
 
 jest.mock('cross-fetch');
+jest.mock('rdf-dereference');
```

Comment:
```suggestion
jest.mock('rdf-dereference', (): any => ({
  dereference: jest.fn(),
}));
```

Reply:
Had this in mind last week. So effectively you are providing an implementation instead of auto-mocking here, if I understand this correctly?

Reply:
Yes. It's a bit of a weird situation since the rdfdereference thing does not seem to be a real class (or something else goes wrong, not sure), so this way we force it to be what we want.